docs/Conformance Criteria/README-Conformance Criteria V2.md

```diff
@@ -0,0 +1,358 @@
+   <th style="background-color:#555555">4</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th></th>
+   <td><b>Support:</b> Extension has documentation with instructions on how to report problems that are related to the extension and not Zowe Explorer. It needs to explain how users can determine if a problem is related to the extension or Zowe Explorer.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">5</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th></th>
+   <td><b>User settings consistency:</b> Extender provides a consistent user settings experience. For VS Code extensions, extender follows the recommended naming convention for configuration settings as described in VS Code's <a href="https://code.visualstudio.com/api/references/contribution-points#contributes.configuration">configuration contribution documentation</a></td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">6</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th></th>
+   <td><b>Using .zowe in user settings:</b> Extender avoids starting setting names with the prefix `zowe.`, which is reserved for Zowe Explorer and and other extensions maintained by the Zowe Project
+.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">7</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th></th>
+   <td><b>Error message consistency:</b> Extension follows the recommended error message format indicated in the <a href=https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/extenders/Error%20Handling.md>Zowe Explorer extensibility documentation</a> to provide a consistent user experience with Zowe Explorer.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">8</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th></th>
+   <td><b>Zowe SDK usage:</b> Extension utilizes the available Zowe SDKs that standardize z/OS interactions as well as other common capabilities that are used by many other Zowe extensions and tools unless the extension's goal is to provide a new implementation with clearly stated goals.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">9</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th></th>
+   <td><b>Sharing of profiles with Zowe CLI:</b> Extensions that utilize Zowe CLI profiles must share the created profile instances between Zowe CLI and the Zowe Explorer extension that utilize them.</td>
```

Comment:
I think we should explicitly mention team configuration.

Reply:
wouldn't `utilize Zowe CLI profiles` cover both the config file as well as the backwards compatibility with V1 style profiles?